specklepy/objects/structural_properties.py

```diff
@@ -0,0 +1,216 @@
+from enum import Enum
+from typing import Any, List, Optional
+
+from .structural_geometry import Axis
```

Comment:
this doesn't work because it is a circular import as `structural_properties` is being wildcard imported into `structural_geometry`. you'll need to reconsider the structure of these modules to fix this